Dockerfile

```diff
@@ -1,12 +1,18 @@
-FROM digitalpulp/front-end:codeship-10
+FROM conchoid/docker-nodenv-builtins
 LABEL maintainer="digitalpulp"
 
+COPY entrypoint.sh /usr/local/bin/
+
+RUN apk add --no-cache make gcc g++ python git && chmod ugo=rx /usr/local/bin/entrypoint.sh
+
+RUN git clone https://github.com/nodenv/node-build-update-defs.git "$(nodenv root)"/plugins/node-build-update-defs && nodenv update-version-defs
```

Comment:
This gets us the build definitions for more node versions than are supplied by the `docker-nodenv-builtins` box.

Reply:
Should we add a list of node versions to the README for which we have build defs.?